specification/billing/resource-manager/Microsoft.Billing/stable/2024-04-01/billingAccount.json

```diff
@@ -0,0 +1,1342 @@
+          "description": "Indicates the reason for the ineligibility of the payment terms.",
+          "enum": [
+            "Other",
+            "OverlappingPaymentTerms",
+            "InvalidDateFormat",
+            "InvalidDateRange",
+            "InactiveBillingAccount",
+            "InvalidBillingAccountType",
+            "NullOrEmptyPaymentTerms",
+            "BillingAccountNotFound",
+            "IneligibleBillingAccountStatus",
+            "InvalidTerms"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PaymentTermsEligibilityCode",
+            "modelAsString": true
+          }
+        },
+        "message": {
+          "description": "Indicates the message for the ineligibility of the payment terms.",
+          "type": "string"
+        }
+      }
+    },
+    "PaymentTermsEligibilityResult": {
+      "description": "Result of the payment terms eligibility.",
+      "type": "object",
+      "properties": {
+        "eligibilityStatus": {
+          "description": "Indicates the eligibility status of the payment terms.",
+          "enum": [
+            "Other",
+            "Valid",
+            "Invalid"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PaymentTermsEligibilityStatus",
+            "modelAsString": true
+          }
+        },
+        "eligibilityDetails": {
+          "description": "Details of the payment terms eligibility.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PaymentTermsEligibilityDetail"
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "RegistrationNumber": {
+      "description": "Describes the registration number of the organization linked with the billing account.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The unique identification number of the organization linked with the billing account.",
+          "type": "string"
+        },
+        "required": {
+          "description": "Identifies if the registration number is required for the billing account.",
+          "type": "boolean",
+          "readOnly": true
+        },
+        "type": {
+          "description": "The types of registration number allowed based on the country of the billing account.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "readOnly": true,
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "TaxIdentifier": {
+      "description": "A tax identifier for the billing account.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The id of the tax identifier.",
+          "type": "string"
+        },
+        "type": {
+          "description": "The type of the tax identifier.",
+          "enum": [
+            "Other",
+            "BrazilCcmId",
+            "BrazilCnpjId",
```

Comment:
>BrazilCnpjId [](http://example.com/codeflow?start=13&length=12)

needs to be added to spelling dictionary assuming its not a typo

Reply:
Added